tests/visual/dropdowns-rtl.html

```diff
@@ -155,6 +155,7 @@
                                 <li kendodropdownsselectable="" role="option" class="k-item k-state-selected k-state-focused"> Focused Selected item </li>
                                 <li kendodropdownsselectable="" role="option" class="k-item k-state-disabled"> Disabled item </li>
```

Comment:
You can clear the tests' markup by removing the `kendodropdownselectable=""` attribute all over the test files